docs/config_search.md

```diff
@@ -147,57 +210,77 @@ profile_models:
-`max_batch_size`) \* (2 `max_queue_delay_microseconds`) \* (2 `instance_group`) values.
+### **Examples of Additional Model Config Parameters**
 
-### Examples of Additional Model Config Parameters
-
-As mentioned in the previous section, manual configuration search allows you to
-sweep on every parameter that can be specified in Triton model configuration. In
-this section, we describe some of the parameters that might be of interest for
+In this section, we describe some of the parameters that might be of interest for
 manual sweep:
 
 - [Rate limiter](https://github.com/triton-inference-server/server/blob/main/docs/user_guide/model_configuration.md#rate-limiter-config) setting
 - If the model is using [ONNX](https://github.com/triton-inference-server/onnxruntime_backend) or [Tensorflow backend](https://github.com/triton-inference-server/tensorflow_backend), the "execution_accelerators" parameters. More information about this parameter is
   available in the [Triton Optimization Guide](https://github.com/triton-inference-server/server/blob/main/docs/user_guide/optimization.md#framework-specific-optimization)
 
-## Quick Search Mode
+---
 
-Quick search can be enabled by adding the parameter `--run-config-search-mode quick` to the CLI.
+## Quick Search Mode
 
-This search mode is the default for ensemble models or when profiling concurrently.
+**Default search mode when profiling ensemble models or multiple models concurrently**
 
-It uses a hill climbing algorithm to search the configuration space, looking for
+This mode uses a hill climbing algorithm to search the configuration space, looking for
 the maximal objective value within the specified constraints. In the majority of cases
 this will find greater than 95% of the maximum objective value (that could be found using a brute force search), while needing to search less than 10% of the configuration space.
 
-After it has found the best config(s), it will then sweep the top-N configurations found (specified by `--num-configs-per-model`) over the default concurrency range before generation of the summary reports.
+After quick search has found the best config(s), it will then sweep the top-N configurations found (specified by `--num-configs-per-model`) over the default concurrency range before generation of the summary reports.
+
+---
+
+_An example model analyzer YAML config that performs a Quick Search:_
+
+```yaml
+model_repository: /path/to/model/repository/
+
+run_config_search_mode: quick
+profile_models:
+  - model_A
+```
+
+---
+
+### **Limiting Batch Size, Instance Count, and Client Concurrency**
+
+Using the `--run-config-search-<min/max>...` config options you have the ability to clamp the algorithm's upper or lower bounds for the model's batch size and instance count, as well as the client's request concurrency.
+
+_Note: That by default quick search runs unbounded and ignores any default values for these settings_
```

Comment:
Could you change this to be a bit more readable? E.g. "Note: By default, quick search runs unbounded and ignores any default values for these settings". I know this verbiage existed before this PR but would be good to fix it going forward.

Otherwise, looks good to me.

Reply:
Done